tests/test_ioc_system.py

```diff
@@ -346,6 +346,9 @@ async def test_create_softioc_record_update_send_to_panda(
     finally:
         m1.close()
 
+    # Give the queue time to be put to
+    await asyncio.sleep(1)
```

Comment:
Notices these tests were failing occasionally because the queue was out of sync so decided to give the ioc process a second to add to parse the command before checking.